config/schemas/config.yaml

```
@@ -5897,6 +5897,14 @@ properties:
                   items:
                     type: string
                   default: []
+                loadBalancerIP:
+                  title: Load Balancer IP
+                  description: |-
+                    Configure the Load Balancer IP from the list of floating ip's created beforehand via openstack cli.
+
+                    This should be enabled when we want to have predefined ip's for the Load Balancer.
```

Making the leading description more generic, and adding specific steps for OpenStack afterwards.
```suggestion
                    Configure the Load Balancer IP to use an existing IP if supported by the infrastructure provider.

                    > [!important]
                    > With OpenStack Octavia the floating IP can be created via the CLI beforehand, and one should set the annotation `loadbalancer.openstack.org/keep-floatingip: "true"` to prevent the floating IP to be deleted.
```